app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -174,7 +170,54 @@ public function getCartDataJs($multipage = true)
 
             // Call Bolt create order API
             try {
-                $orderCreationResponse = $this->createBoltOrder($quote, $multipage);
+                /////////////////////////////////////////////////////////////////////////////////
+                // We create a copy of the quote that is immutable by the customer/frontend
+                // Bolt saves this quote to the database at Magento-side order save time.
+                // This assures that the quote saved to Magento matches what is stored on Bolt
+                // Only shipping, tax and discounts can change, and only if the shipping, tax
+                // and discount calculations change on the Magento server
+                ////////////////////////////////////////////////////////////////////////////////
+
+                /*********************************************************/
+                /* Clean up resources that may have previously been saved
+                /* @var Mage_Sales_Model_Quote[] $expired_quotes */
+                $expired_quotes = Mage::getModel('sales/quote')
+                    ->getCollection()
+                    ->addFieldToFilter('parent_quote_id', $quote->getId());
```

Comment:
I don't see where this column has been added to the sales_flat_quote_table. Are you missing an installer script or am I just not seeing all of the files?

Reply:
@johnmccombs1 Correct.  The install scripts were not in the commit.  They are included now.